Solutions/Marain.UserNotifications.Storage.AzureStorage/Marain/UserNotifications/Storage/AzureStorage/Internal/UserNotificationTableEntity.cs

```diff
@@ -95,5 +95,22 @@ public UserNotification ToNotification(JsonSerializerSettings serializerSettings
                 new UserNotificationMetadata(correlationIds, this.ETag),
                 channelDeliveryStatuses);
         }
+
+        private static string GetHexaDecimalString(byte[] identityHash)
+        {
+            // Create a new Stringbuilder to collect the bytes
+            // and create a string.
+            var sBuilder = new System.Text.StringBuilder(64);
```

Comment:
Better to use `identityHash.Length * 2` instead of 64 for two reasons:

1. it makes it clear where the number comes from (otherwise it's just a magic number without context)
2. if for some reason we were to change the hash type to a different length, it would still be correct

Reply:
Hi Ian, thanks for the review, I have made the changes and pushed them now.